### PR TITLE
consistent jump height 

### PR DESCRIPTION
**Pull request by norgeous**

### CODE REVIEW

File: src/objects/entities/Ben3.ts [hunk 1]
```diff
@@ -107,10 +107,19 @@ class Ben3 extends Entity {
...
-      this.gameObject.applyForce({ x: 0, y: -0.3 });
-      // @ts-expect-error todo
-      this.head.applyForce({ x: 0, y: -0.3 });
+      const { body: Body } = this.scene.matter;
+
+      const { centerX, centerY } = this.gameObject.getBounds();
+      const position = { x: centerX, y: centerY };
+      Body.applyForce(this.gameObject.body, position, {
+        x: 0,
+        y: -0.05 * this.gameObject.body.mass,
```
Comment (ScottyXIII) [inline src/objects/entities/Ben3.ts:116]: Did we not want to multiply this by the mass of the head or does the body include head mass?

Comment (norgeous) [reply src/objects/entities/Ben3.ts:116]: force is applied to head on line 119

its the same force, which is 5% of the objects mass

File: src/scenes/game-scene.ts [hunk 2]
```diff
@@ -109,17 +109,22 @@ const soundConfig = [
     filePath: './audio/music/spook.mp3',
     loop: true,
   },
+  {
+    key: 'jump',
+    filePath: './audio/sfx/jump.mp3',
+    loop: false,
+  },
```
Comment (ScottyXIII) [inline src/scenes/game-scene.ts:116]: Nice 👍 I'm wondering if we have options to toggle soundFx/music on/off now 🤔

**Review by ScottyXIII — APPROVED** [review]

File: src/scenes/game-scene.ts (hunk 2 above)
Comment (norgeous) [reply src/scenes/game-scene.ts:116]: like separate controls for sfx vs music?
yeah its defos something ive seen and used before
like if i want to play a game, but am sick of the music they have, i can just get sfx from the game and play my own music
i like it

Comment (norgeous) [reply src/scenes/game-scene.ts:116]: lets make a card in trello for it
would be a good for next jam if we find we dont have time to do this idea

**Pull request merged**